delphi/translators/for2py/floatNumpy.py

```diff
@@ -0,0 +1,149 @@
+from numpy import float32
+
+################################################################################
```

Comment:
To be consistent with the coding style used in other files in this system, this line (and a similar line later in this file) should be deleted.